syntax = "proto3";

package api;

// Define a message for the Product model
message Inventory {
  string id = 1;
  string item_name = 2;
  string description = 3;
  float unit_price = 4;
  int32 stock_in_hand = 5;
  int32 threshold = 6;
  string email = 7;
}

// Define a message for the operations on the Product
message InventoryUpdates {
  string operation = 1;
  Inventory data = 2;
}

//protoc --python_out=. inventory.proto
